src/celeritas/ext/GeantPhysicsOptions.hh

```diff
@@ -0,0 +1,70 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2022 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/ext/GeantPhysicsOptions.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "celeritas/Quantities.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+//! Brems selection (TODO: make bitset)
```

Comment:
I noticed the use of bit flags in `GeantImporter.hh` and wanted to ask: What is the real advantage of setting up `0xN` enums?

Reply:
`0xN` is a canonical way of defining bit flags in C and C++ since it's easier to see that `0x00400000u` has only one bit set (compared to `4194304`). In our case we're only using a few bits so it in practice doesn't matter, but I find it to be a nice way of "annotating" that these values aren't meant to be consecutive integers but rather exclusive bit patterns. The "best" way of doing this is to have a code abstraction that lets us turn consecutive enums into a nice bitfield, but I didn't think it worth the effort for this use case so far.